go/vt/vttablet/onlineddl/executor.go

```diff
@@ -3234,7 +3238,7 @@ func (e *Executor) runNextMigration(ctx context.Context) error {
 		if err != nil {
 			return nil, err
 		}
-		r, err := e.execQuery(ctx, sqlSelectReadyMigrations)
+		r, err := e.execQuery(ctx, fmt.Sprintf(sqlSelectReadyMigrations, sidecardb.GetIdentifier()))
```

Comment:
can we ensure the query always executes on the correct schema (ie the keyspace's sidecardb) rather than re-inject the schema name in all queries?